biojava-structure/src/main/java/org/biojava/nbio/structure/asa/AsaCalculator.java

```diff
@@ -135,7 +165,7 @@ static class IndexAndDistance {
 	 * NACCESS' -h option
 	 */
 	public AsaCalculator(Structure structure, double probe, int nSpherePoints, int nThreads, boolean hetAtoms) {
-		this.atoms = StructureTools.getAllNonHAtomArray(structure, hetAtoms);
+		this.atoms = StructureTools.getAllNonHAtomArray(structure, hetAtoms, 1);
```

Comment:
Delegating to the new constructor makes this easier to maintain.

```suggestion
		this(structure, probe, nSpherePoints, nThreads, hetAtoms, 1);
```